tst/performance/test_meshblock_data_iterator.cpp

```diff
@@ -74,21 +76,29 @@ void performance_test_wrapper(const std::string &test_name, InitFunc init_func,
 
 static MeshBlockData<Real> createTestContainer() {
   // Make a container for testing performance
-  MeshBlockData<Real> container;
   std::vector<int> scalar_shape{N, N, N};
   std::vector<int> vector_shape{N, N, N, 3};
 
   Metadata m_in({Metadata::Independent}, scalar_shape);
   Metadata m_in_vec({Metadata::Independent}, vector_shape);
 
   // make some variables - 5 in all, 2 3-vectors, total 10 fields
-  container.Add("v0", m_in);
-  container.Add("v1", m_in);
-  container.Add("v2", m_in_vec);
-  container.Add("v3", m_in);
-  container.Add("v4", m_in_vec);
-  container.Add("v5", m_in);
-  return container;
+  auto pkg = std::make_shared<StateDescriptor>("Test package");
+  pkg->AddField("v0", m_in);
+  pkg->AddField("v1", m_in);
+  pkg->AddField("v2", m_in_vec);
+  pkg->AddField("v3", m_in);
+  pkg->AddField("v4", m_in_vec);
+  pkg->AddField("v5", m_in);
+
+  // we need to connect the MeshBlockData to a dummy mesh block, otherwise variables
+  // won't be allocated
+  auto dummy_mb = std::make_shared<MeshBlock>(16, 3);
+
+  MeshBlockData<Real> rc;
+  rc.Initialize(pkg, dummy_mb);
+
+  return rc;
```

Comment:
```suggestion
  MeshBlockData<Real> mbd;
  mbd.Initialize(pkg, dummy_mb);

  return mbd;
```
We still have plenty of `rc` in the code for the `Container<Real>`  that don't exist in their form any more.
So whenever we update files, we could also update variable names so that new user (who may not be aware of the historic reasons for the `rc` variable name) don't get confused.

Reply:
Also, this may be a "good first issue" in terms of cleanup up code.